rundnsmasq.sh

```diff
@@ -17,7 +22,7 @@ mkdir -p /shared/log/dnsmasq
 cp /usr/share/ipxe/undionly.kpxe /usr/share/ipxe/ipxe.efi /shared/tftpboot
 
 # Use configured values
-sed -i -e s/IRONIC_IP/${PROVISIONING_IP}/g -e s/HTTP_PORT/${HTTP_PORT}/g \
+sed -i -e s/PROVISIONING_IP/${PROVISIONING_IP}/g -e s/HTTP_PORT/${HTTP_PORT}/g \
```

Comment:
This change doesn't look right

Reply:
I checked /etc/dnsmasq.conf and it looks like it still expects IRONIC_IP to be substituted

Reply:
Thanks, that's a sed mistake, now fixed!